frontend/src/Components/Events/EventCard.svelte

```diff
@@ -0,0 +1,134 @@
+<script>
+    import { onMount } from 'svelte';
+    import { getEvents } from "./eventstore";
+  
+    let events = [];
+
+    
+    onMount(async () => {
+      // Fetch events from the API and wait for the promise to resolve
+      const allEvents = await getEvents();
+      // Limit the number of events to 3
+      events = allEvents.slice(0, 3);
```

Comment:
Could you limit to like 10 events? also we should sort by start date, and give the 10 next upcoming events cause right now we only get old events. This could be something you do in the backend or the frontend